src/Angular.js

```diff
@@ -1602,6 +1622,11 @@ function angularInit(element, bootstrap) {
     }
   });
   if (appElement) {
+    if (!isAutoBootstrapAllowed) {
```

Comment:
I didn't do this intentionally - otherwise you'll get an error message even if you intend to manually bootstrap by calling `angular.bootstrap`. Which I guess is a legit use case, if somebody does inject Angular in the page but also controls it to call `angular.bootstrap` (not a security issue because they can already execute JS in the page in that case).